agave_app/GLView3D.cpp

```diff
@@ -32,16 +32,18 @@
 #pragma warning(disable : 4351)
 #endif
 
-GLView3D::GLView3D(QCamera* cam, QRenderSettings* qrs, RenderSettings* rs, QWidget* parent)
+GLView3D::GLView3D(QRenderSettings* qrs, RenderSettings* rs, QWidget* parent)
   : QOpenGLWidget(parent)
   , m_etimer()
-  , m_qcamera(cam)
   , m_viewerWindow(nullptr)
   , m_qrendersettings(qrs)
 {
   m_viewerWindow = new ViewerWindow(rs);
   m_viewerWindow->gesture.input.setDoubleClickTime((double)QApplication::doubleClickInterval() / 1000.0);
 
+  // camera is created deep down inside m_viewerWindow.
+  m_cameraDataObject = new CameraDataObject(&m_viewerWindow->m_CCamera);

```

Comment:
initialization order for cameras is now slightly different so we can introduce the CameraDataObject from which the gui will be constructed.